src/Language/Fortran/Parser/Fortran77.y

```diff
@@ -136,6 +169,24 @@ import Debug.Trace
 
 %%
 
+maybe(p)
```

Comment:
turns out recent versions of happy *do* support parameterized productions! 

https://www.haskell.org/happy/doc/html/sec-grammar.html